### PR TITLE
distribution: Weekly team updates

### PR DESCRIPTION
**Pull request by pecigonzalo**
While the document describes updates as a bi-weekly process, we are effectively doing weekly updates. Bi-weekly would be too infrequent given we would only get 2 updates on each iteration. The reasons for async and weekly are still valid from the bi-weekly ones, so I kept them.

> - Weekly updates make it easy to forget what you did at the start of the week, forget to ask for help, and make it hard for others to offer help in a timely fashion.

~I have removed this point as it was not clear to me "why" weekly updates would cause this.~

### CODE REVIEW

**Comment by bobheadxi** [comment]
> I have removed this point as it was not clear to me "why" weekly updates would cause this.

I believe this means "by friday I might not remember what I did on monday" (ie on wednesday you are more likely to reflect on what you did on monday and ask for help early)

**Comment by bobheadxi** [comment]
I personally quite like having to reflect a bit mid-week and read some updates from everyone 🤔 I would advocate to keep our current bi-weekly format (light one on wednesday, proper tracking issue writeup on friday)

**Comment by pecigonzalo** [comment]
@bobheadxi we are still having 2 updates per week. Maybe I'm confused here, but as I understand it, bi-weekly means it happens every 2 weeks, not that we have 2 updates per week.

**Comment by pecigonzalo** [comment]
I just saw your update, I was not aware that `bi-weekly` could also be used as "twice a week".  I would prefer to rephrase it to make it more clear then.

**Comment by pecigonzalo** [comment]
I have updated the docs adding the section back, but rewording. Bi-weekly seems to be a more confusing term than "twice a week", let me know if its more clear now.

Thanks @bobheadxi

File: handbook/engineering/distribution/recurring_processes.md [hunk 1]
```diff
@@ -188,7 +188,7 @@ Wed, before EOD (local time): distribution members are expected to post an updat
 
 The goal of this update is to ensure we're discussing things as a team, asking for help when appropriate, reflecting on our progress, and giving others the opportunity to provide help and guidance.
 
-#### Friday update
+#### Friday Github update
```
Comment (bobheadxi) [inline handbook/engineering/distribution/recurring_processes.md:191]:
```suggestion
#### Friday GitHub update
```
